### ISSUE TITLE
Plataforma-Proveedor-Subir CFDI-Oatsop

### ISSUE THREAD

**Issue by RigoFlores** [jccastillo0007/eFacturaT#687]
Envíar la notificación del proveedor, solo en el caso de haber subido exitosamente un CFDI.

NO ENVIAR NOTIFICACIÓN SI:
a) El CFDI no superó validación del SAT
b) El CFDI ya está duplicado (ya existía o lo habían subido anteriormente anteriormente)

**Comment by jccastillo0007** [comment]
cierto... estaba en un lugar donde siempre lo mandaba, para probar el servicio, tons ya lo pongo como debe ser.
Listo en Pru.

**Comment by RigoFlores** [comment]
Ahora ya no envía nada... 
Será que ya se acabó nuestro presupuesto?

**Comment by jccastillo0007** [comment]
Debe enviarlo cada que subes una factura que si esta vigente y todo esta correcto, aún asi no la esta enviando ?

cuantos mensajes llevas ?
